operations/settings/base.py

```diff
@@ -113,15 +115,31 @@
             'formatter': 'verbose',
             'level': 'ERROR',
         },
+        'sentry': {
+            'level': 'ERROR',
+            'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler',
+            'tags': {'custom-tag': 'x'},
+        },
     },
     'root': {
-        'handlers': ['console'],
+        'handlers': ['sentry', 'console'],
         'level': 'INFO',
         'propagate': False,
     },
     'version': 1,
 }
 
+# ######### RAVEN CONFIG
+
+RAVEN_CONFIG = {
```

Comment:
since dev doens't have raven, can we push all the Raven stuff to `production.py`